gruntfileConfig.json

```diff
@@ -4,6 +4,7 @@
   "src": "./src",
   "assets": "./src/assets",
   "theme": "./node_modules/patternpack-example-theme",
+  "logo": "/theme-assets/images/pp-logo.svg",
```

Comment:
Reading the documentation, it appears that you are recommending an image from the normal assets directory.  This would then be the default configuration for the example-theme.

This then opens the following question:  If I am an author of a theme, I must now choose
 1. Use `pp-logo.png` as my logo file name
 1. Use my own name and force every consumer of the theme to specify my default name
 1. Hardcode my own logo name and lose the ability to override

I'm not necessarily opposed to this.  I just wanted to explore the consequences of the current design...